contract/src/orca.proposal.js

```diff
@@ -55,21 +76,29 @@ export const startOrcaContract = async (permittedPowers, config) => {
 
   trace('config', config);
   trace('permittedPowers', permittedPowers);
-  const {
-    // must be supplied by caller or template-replaced
-    bundleID = Fail`no bundleID`,
-  } = config?.options?.[contractName] ?? {};
-
-  // agoricNames gets updated each time; the promise space only once XXXXXXX
-
-  const installation = await installContract(permittedPowers, {
-    name: contractName,
-    bundleID,
-  });
```

Comment:
Taking this out isn't compatible with `runCoreEval` from e2e-tools. I wonder how the `Start Orca contract using core-eval` test is passing.